README.md

```diff
@@ -982,7 +982,7 @@
     // declaration to the top of the scope,
     // which means our example could be rewritten as:
     function example() {
-      let declaredButNotAssigned;
+      var declaredButNotAssigned;
```

Comment:
right - but since `let declaredButNotAssigned` is now *at the top of the block*, it matches the semantic of the var in the previous snippet.